nototools/android_patches.py

```diff
@@ -114,17 +121,47 @@ def _remove_from_cmap(infile, outfile, exclude=[]):
       + [0x1F238, 0x1F239, 0x1F23A, 0x1F250, 0x1F251]
   )
 
-  names = ['cjk/NotoSans%sCJK%s-Regular.otf' % (m, v)
-           for m in ['', 'Mono']
-           for v in ['jp', 'kr', 'sc', 'tc']]
-  for font_name in names:
+  for font_name in cjk_font_names:
     print 'remove cjk emoji', font_name
     _remove_from_cmap(
-        font_name,
-        path.join(outdir, font_name),
+        path.join(srcdir, font_name),
+        path.join(dstdir, font_name),
         exclude=EMOJI)
 
 
+def patch_cjk_ttc(ttc_srcfile, ttc_dstfile):
```

Comment:
CrOS may need different changes than Android. Not now, but eventually we may have to move this and other 'generic' functions (not tied to Android) out of android-patches.py.